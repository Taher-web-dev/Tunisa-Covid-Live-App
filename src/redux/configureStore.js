import { createStore, combineReducers, applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
import updateReducer from './LATEST/reducer/reducer';

const reducer = combineReducers({
  latest: updateReducer,
});

const store = createStore(
  reducer,
  applyMiddleware(thunk, logger),
);
