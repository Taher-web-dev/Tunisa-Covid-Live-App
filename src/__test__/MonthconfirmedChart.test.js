import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import MonthConfirmedChart from '../components/ConfirmedCases/ChartConfirmedMonth';

const mockStore = configureStore([]);

describe('render correctly MonthConfirmedChart component', () => {
  let store;
  let tree;
  beforeEach(() => {
    store = mockStore({
      latest: {},
      dayBefore: {},
      historical: {},
    });
    tree = renderer.create(
      <Provider store={store}>
        <MonthConfirmedChart />
      </Provider>,
    );
  });
  it('should render with given state from redux store', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
