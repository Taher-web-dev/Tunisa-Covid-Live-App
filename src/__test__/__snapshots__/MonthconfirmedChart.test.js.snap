// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly MonthConfirmedChart component should render with given state from redux store 1`] = `
<div
  style={
    Object {
      "height": "200px",
      "paddingBottom": "20px",
    }
  }
>
  <canvas
    height={150}
    role="img"
    width={300}
  />
</div>
`;
